musicround/helpers/import_helper.py

```
@@ -29,6 +34,111 @@ def generate_token(length=32):
+        # However, for consistency, it's better if the caller (import_spotify_track) always passes it.
+        token_to_use = token
+        if not token_to_use:
+            expires_at_timestamp = None
+            if current_user.spotify_token_expiry:
+                if isinstance(current_user.spotify_token_expiry, datetime):
+                    expires_at_timestamp = int(current_user.spotify_token_expiry.timestamp())
+                else:
+                    try:
+                        expires_at_timestamp = int(datetime.fromisoformat(str(current_user.spotify_token_expiry)).timestamp())
+                    except ValueError:
+                        pass # Logged by caller
```

Ruff (0.11.9)

Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

(SIM105)